### PR TITLE
Hide multipage nav (secondary nav) if there's no links to show

### PR DESCRIPTION
**Pull request by mmmavis**
- Hide multipage nav (secondary nav) if there's no links to show (#10919)
- Hide FA's .wFormFooter and .supportInfo (as mentioned in [#10920](https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10920#issuecomment-1648085670)) Note that the remaining tasks in #10920 will be tackled in new PRs.

### Test pages
- test page 1 (no secondary nav): https://foundation-s-10920-peti-ejxqei.herokuapp.com/en/campaigns/single-page/
- test page 2 (with secondary nav): https://foundation-s-10920-peti-ejxqei.herokuapp.com/en/campaigns/multi-page/

---

Tagging you @kristinashu as I'm unsure which designer I should assign this PR review to 😅

### CODE REVIEW

File: source/js/multipage-nav.js [hunk 1]
```diff
@@ -1,54 +1,70 @@
 /**
  * Inject mobile version of multipage navs (secondary navs)
- *
```
Comment (mmmavis) [inline source/js/multipage-nav.js:3]: A lot of changes in this file are just white space changes.

**Review by kristinashu — APPROVED** [review]
Much better, thank you Mavis!

**Review by danielfmiranda — COMMENTED** [review]
Hi @mmmavis thanks for taking on this work! 

After testing this out, I can confirm that everything is working as expected. The grey bar is gone, and both divs have been hidden.

However, I do have one question:

Regarding the .formFooter and .supportInfo divs that are now being set to `display:none;`, will these divs **always** be empty? Or, do users have the option of adding content within these divs through FormAssembly?

If so, I am concerned that permanently adding the property `display:none;` to these divs through SCSS will hide elements that are meant to be seen.

Thanks!

**Comment by mmmavis** [comment]
> Regarding the .formFooter and .supportInfo divs that are now being set to display:none;, will these divs always be empty? Or, do users have the option of adding content within these divs through FormAssembly?

@danielfmiranda thanks for the q. Both sections will always be empty!

**Review by danielfmiranda — APPROVED** [review]
Thanks for the clarification Mavis! Approved 🙌

**Pull request merged**